cmd/cephcsi.go

```diff
@@ -177,13 +171,13 @@ func getDriverName() string {
 	}
 	// select driver name based on volume type
 	switch conf.Vtype {
-	case rbdType:
+	case util.RBDType:
```

Comment:
The commit message has a type. Why is this needed/useful? **THAT** should be included in the commit message.